libs/gretty/src/main/groovy/org/akhikhl/gretty/GrettyPlugin.groovy

```diff
@@ -263,11 +263,11 @@ class GrettyPlugin implements Plugin<Project> {
         into "${project.buildDir}/inplaceWebapp"
       }
 
-      project.task('createInplaceWebAppFolder', group: 'gretty') {
-
+      project.tasks.register('createInplaceWebAppFolder') {
+        group = 'gretty'
         description = 'Creates directory ${buildDir}/inplaceWebapp'
 
-        dependsOn project.tasks.prepareInplaceWebAppFolder
```

Comment:
When in a `project.tasks.register` block, there is no need to use `tasks.named` inside. The block is "lazy" anyways so this change (and all the similar ones in this PR) could be reverted.

See [here](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html#sec:task_configuration_avoidance_pitfalls).

Reply:
OK will do.